Documentation/install/bare-metal/requirements.md

```diff
@@ -130,3 +130,4 @@ The provisioner must:
 [network-setup]: https://coreos.com/matchbox/docs/latest/network-setup.html
 [dns]: index.md#dns
 [using-f5]: ../../reference/f5-ltm-lb.md
+[release-notes]: https://github.com/coreos/tectonic-installer/blob/master/changes.md
```

Comment:
tectonic release notes get published on the coreos.com site at https://coreos.com/tectonic/releases/, and you can link to individual versions in that list like so: https://coreos.com/tectonic/releases/#1.7.3-tectonic.3

Reply:
this is for the installer - you want to link to the regular notes for this?

Reply:
ah - they're non-sequential on the site. i see it. will do.